src/AutoRest.CSharp/LowLevel/Generation/LowLevelDataPlaneClientWriter.cs

```diff
@@ -317,12 +319,13 @@ private void WriteClientMethodBody(CodeWriter writer, LowLevelClientMethod clien
         private void WriteClientMethodDecleration(CodeWriter writer, RestClientMethod clientMethod, LowLevelOperationSchemaInfo operationSchemas, bool async)
         {
             var parameters = clientMethod.Parameters.Concat(new Parameter[] { RequestOptionsParameter });
+            var isConstantResponseBody = clientMethod.Responses.All(response => response.ResponseBody is ConstantResponseBody body);
```

Comment:
Why `clientMethod.ReturnType == typeof(bool)` doesn't work here?

Reply:
Added this condition.